src/SFA.DAS.LevyTransferMatching.Web/Models/Pledges/GetApplicationViewModel.cs

```diff
@@ -1,12 +1,30 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Globalization;
+using System.Linq;
 using SFA.DAS.LevyTransferMatching.Infrastructure.Dto;
+using SFA.DAS.LevyTransferMatching.Infrastructure.ReferenceData;
+using SFA.DAS.LevyTransferMatching.Web.Extensions;
+using StructureMap.Query;
 
 namespace SFA.DAS.LevyTransferMatching.Web.Models.Pledges
 {
-    public class ApplicationViewModel
+    public class GetApplicationViewModel
     {
+        public GetApplicationViewModel()
+        {
+            
+        }
+
+        public void SetupModel()
```

Comment:
I don't think we need this - can't we just set the `DisplaySectors` property in the orchestrator when at the same time as all the other properties?

Reply:
yeah, not sure how I didn't realise this.